data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -75,6 +76,37 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
         { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200, "bytes": 1000, "http_verb": "GET" }
       ```
 
+### <a name="count"></a>
+* `count`: Count Events belonging to the same group and generate a new event with values of the identification keys and the count, indicating the number of events. All Events that make up the combined Event will be dropped.
+    * It supports the following config options
+       * `countKey`: key name to use for storing the count, default name is "aggr._count"
+       * `outputFormat`: format of the aggregated event.
```

Comment:
Sorry I missed this on last review. We have opted for snake case variables in the pipeline configurations. These should be `count_key` and `output_format`